### ISSUE TITLE
Devtools installation issue

### ISSUE THREAD

**Issue by rberg2** [GraceYoon/SPRING#3]
Hello,
I am running into this error while trying to install SPRING. This is running R 4.1.2 and inside a docker container so sometimes things break due to how minimal the container is.

```
> devtools::install_github("GraceYoon/SPRING")
Downloading GitHub repo GraceYoon/SPRING@HEAD
Error: Failed to install 'SPRING' from GitHub:
  Line starting 'LazyData ...' is malformed!
Execution halted
```

Do you have any advice?
Thanks!

**Comment by irinagain** [comment]
I've just tried on Mac OS with R 4.1.2, and I can't reproduce, unfortunately. I have no issues with installation. I have tried googling, and it seems that this issue occasionally comes up (with other packages from Github) but is hard to track or reproduce. This is the closest I found to a possible solution: [StackOverflow link](https://stackoverflow.com/questions/68332879/failed-to-download-a-package-from-github-in-rstudio)
